zh_cn/downloads/index.md

```diff
@@ -4,106 +4,58 @@ title: "下载 Ruby - Ruby 官方网站"
+* Linux/UNIX 平台，可以使用第三方工具（如 rbenv 或 RVM）或使用系统包管理工具。
+* OS X 平台，可以使用第三方工具（如 rbenv 或 RVM）。
+* Windows 平台，可以使用 RubyInstaller。
 
-更多信息见 [Ruby Core](/zh_cn/community/ruby-core/) 页。
+进一步了解请了解[安装][installation]页面上关于使用包管理工具与第三方工具的内容。
 
-### 镜像站
+当然，每个平台都可以从源代码来安装 Ruby。
 
-在世界各地的[镜像站][mirrors]上面已经有 Ruby 的源代码了，你可以选择就近的地方下载。
+### 从源代码编译 Ruby
 
-### Windows 系统
+如果您非常熟悉您的操作系统，或是您的环境需要特別设定，那么直接使用源代码来安装是个极佳的方式。如果您的平台没有编译好的包的话，则需要通过源代码来安装。
 
-在 Windows 平台下安装 Ruby
-有几个选择。第一个选择是仅安装编译好的二进制文件。第二个选择是直接执行“一步安装”程序，假如您不知道如何安装
-Ruby,[RubyInstaller][5]将是您最好的选择。（这种安装方式除 Ruby 之外，捆绑一些额外的资源库。）
+进一步关于从源代码编译 Ruby 的内容，请参考[安装][installation]页面。若编译 Ruby 时遇到任何问题，请参考安装页面罗列的第三方工具，可能会有帮助。
 
-* [Ruby 1.8.6 一步安装][6] (md5:&nbsp;00540689d1039964bc8d844b2b0c7db6) 稳定版
-  (*推荐*)
-* [Ruby 1.8.6 二进制][7] (md5:&nbsp;3ec2a8a34d5db1f09cc8cad3f8305c28) 稳定版 (*推荐*)
-* [Ruby 1.9.0 二进制][8] (md5:&nbsp;489ee1bcd72e97f50b38b64fb354e661) 开发版 (*实验*)
-* [Ruby 1.9.1-p0 二进制][9] (md5:&nbsp;921db53c709ce20106610f7d229e24e1) 稳定版
-  (*推荐*)
+* **当前稳定版：**
+  [Ruby {{ site.downloads.stable.version }}][stable-gz]<br>
+  sha256: {{ site.downloads.stable.sha256.gz }}
 
-### Linux 系统
+* **上一个稳定版：**
+  [Ruby {{ site.downloads.previous.version }}][previous-gz]<br>
+  sha256: {{ site.downloads.previous.sha256.gz }}
 
-许多 Linux 发行版中已经预装了 Ruby，如果您的系统上没有安装
-Ruby，这里提供几个方式来安装。第一个方式是仅下载源代码，然后手工编译安装。
-如果您不想从源代码编译浪费时间的话，许多平台都提供相应的机器体系结构的发行包，它们使 Ruby 的安装相当容易。
+* **旧稳定版（2.0.0 系列）：**
+  [Ruby {{ site.downloads.previous20.version }}][previous20-gz]<br>
+  sha256: {{ site.downloads.previous20.sha256.gz }}
 
-例如，在 Debian 或 Ubuntu 使用 `apt-get` 提供了一个简单而优雅的解决方案 ：
+* **快照：**
+  * [Stable Snapshot][stable-snapshot-gz]:
+    当前稳定版 tarball 的最新快照
+  * [Nightly Snapshot][nightly-gz]:
+    这是 SVN 上的 tarball，每晚更新。
+    可能有问题或 bug，谨慎使用！
 
-{% highlight sh %}
-$ sudo apt-get install ruby irb rdoc
-{% endhighlight %}
+关于 Ruby Subversion 与 Git Repositories 的信息，参见 [Ruby Core](/en/community/ruby-core/) 页面。
 
-对于 irb 和 rdoc ，您需要激活 universe repository 。
-
-### OS X 系统
-
-Mac OS X Lion 完全支持 Ruby 1.8.7 以及许多常见的 Ruby gems (packages)。细节参考 [Ruby
-wiki at MacOS Forge][10] 。
-
-Mac OS X Leopard 完全支持 Ruby 1.8.6 以及 Ruby on Rails, Mongrel, Capistrano
-和其他流行的 Ruby gems （库）。细节请参考 [Ruby MacOS Forge 上的 Ruby 维基][10] 。
-
-Mac OS X Tiger 安装了 1.8.2 版本的 Ruby，如果您没有升级到 Leopard，Tiger 平台下安装最新版本的 Ruby
-仍有多个选择。您可以使用为 OS X 定做的[一步安装][11] 。如果您正在寻找且能够快速运行 Rails
-环境的安装包，[Locomotive][12] 是一个不错的选择。使用 [MacPorts][13] 或 [Fink][14]
-在技术上的可能会比较好一些。
-
-使用 MacPorts，您可以这样来安装 Ruby …
-
-{% highlight sh %}
-$ port install ruby
-{% endhighlight %}
-
-Fink 为安装 Ruby 提供了图形接口（使用 Fink 命令）。
-
-同时，由于 OS X 是基于 Unix，因此从源代码或采取其它有效的方法来安装也是相当的容易。
-
-更为详细的安装 Ruby 及 Rails，详见 Dan Benjamin 写的优秀文章： [*Building Ruby, Rails,
-LightTPD, and MySQL on Tiger*][15] 将快速的教您启动和运行。
+Ruby 源代码可从世界各地的[镜像站][mirrors]获得。请尝试离您最近的镜像站。
 
 
 
 [license]: {{ site.license.url }}
-[stable-gz]:   {{ site.downloads.stable.url.gz }}
+[installation]: /zh_tw/documentation/installation/
```

Comment:
This is because the page hasn't been translated. For this kind of occasion, link back to official `en` then! Thank you!